action.sh

```diff
@@ -42,7 +42,17 @@ echo "$long_line"
 echo "${bold}→ Checking datasets${reset}"
 echo "  opensafely check"
 echo
-opensafely check
+if ! opensafely check; then
+  curl \
+        -X POST \
+        --silent \
+        -H "Accept: application/vnd.github.v3+json" \
+        -H "$auth_header" \
+        https://api.github.com/repos/$GITHUB_REPOSITORY/issues \
+        -d "{\"body\":\"Dataset check fail @amirmehrkar at $GITHUB_SHA\", \"title\":\"Dataset check fail\"}" \
```

Comment:
thought (non-blocking): Would this be better as an opensafely organisational group that Amir is a member of, as opposed to his username? Reasons:

* Other people might want to be notified in future, which would mean changing this line.
* A user could change their GitHub username, and this would then silently break, although unlikely.

Reply:
I agree, I don't know enough about github organisational groups but that seems far more sensible than a single-username tag.

Reply:
Changed to @data-access